chessington/ui/static/scripts/board.js

```diff
@@ -75,6 +78,13 @@ export class Board {
     return this.squares[i][j].piece
   }
 
+  // Returns standard chess notation position for the given row/col
```

Comment:
Not sure I can justify why exactly I've chosen to go with chess notation over row column other than having recently watched Queen's Gambit, but we can switch to row/col if we think that would be better
(I'm not too concerned given this is a "stretch" goal anyway)